vnext/Desktop.IntegrationTests/React.Windows.Desktop.IntegrationTests.vcxproj

```diff
@@ -41,7 +41,7 @@
   <PropertyGroup Label="UserMacros" />
   <PropertyGroup>
     <IncludePath>$(ReactNativeWindowsDir);$(ReactNativeWindowsDir)Common;$(ReactNativeWindowsDir)stubs;$(FollyDir);$(ReactNativeDir)\ReactCommon;$(JSI_Source);$(ReactNativeWindowsDir)ReactWindowsCore;$(ReactNativeWindowsDir)include\ReactWindowsCore;$(ReactNativeWindowsDir)Desktop;$(ReactNativeWindowsDir)IntegrationTests;$(MSBuildProjectDirectory);$(IncludePath)</IncludePath>
-    <LinkIncremental>true</LinkIncremental>
+    <LinkIncremental>false</LinkIncremental>
```

Comment:
Whats up with turning off incremental linking?  Esp for test binaries wouldn't we rather have the quicker builds?
For the shipping binaries, it makes sense to turn it off on release builds certainly. -- Maybe we can leave it on in debug?

Reply:
Oh I see your comment about LTCG....

Reply:
Typical VS project usually has no LTCG for debug builds and thus uses incremental linking.
I am not sure why these settings are config-independent.
A better fix would be to have debug-only and release-only config as default VS projects do.

---
In reply to: [364365070](https://github.com/microsoft/react-native-windows/pull/3847#discussion_r364365070) [](ancestors = 364365070)